proto/event/domain/event.proto

```diff
@@ -258,6 +259,11 @@ message FeatureCreatedEvent {
   repeated bucketeer.feature.Target targets = 12;
 }
 
+message FeatureUpdatedEvent {
+  string id = 1;
+  string data = 2;
```

Comment:
`data` contains a JSON-formatted feature object. We can show diffs between the later and previous data by comparing this data.

Reply:
```suggestion
  string data = 2;
  string prev_data = 3;
```